explore/src/main/scala/explore/Explore.scala

```diff
@@ -207,7 +207,7 @@ object ExploreMain extends IOApp.Simple {
 
     (for {
       dispatcher <- Dispatcher[IO]
-      worker     <- WebWorkerF[IO](WebWorkers.TestWorker(), dispatcher)
+      worker     <- WebWorkerF[IO](WebWorkers.CacheIDBWorker(), dispatcher)
```

Comment:
Are we planning to have more workers? If so, maybe we can give this a more specific name.

Reply:
Maybe. I called it `CatalogWorker` originally but then became more general. Given the extra size we get per worker it could make sense to have only one, though for expensive actions (Semester skyplots) maybe we want separte ones